docs/row.checkable.md

```diff
@@ -1,24 +1,41 @@
 # qgrid checkable row
+## descrition
+Checkable rows is just a one of the selection modes we offer to user.
+
+Here is a list of features we should support:
+* Extra column with checkboxes for each row
+* Select all checkbox in the header of the column
+* Column and header should be templatable
+* Selected items should be bound to `selection`
+
+When qgrid works in server side pagination mode we might face a trouble with `selection`.
+Just a simple case:
```

Comment:
Also we do some work on groupbox directives please review it https://github.com/klumba12/groupbox